packages/api/src/index.ts

```diff
@@ -16,7 +16,16 @@ export type {
   TrustSetFlags
 } from '@gemwallet/constants';
 
-// API return types
+// API request types
+export type {
+  GetNetworkRequest,
+  GetNFTRequest,
+  SendPaymentRequest,
+  SetTrustlineRequest,
+  SignMessageRequest
+} from '@gemwallet/constants';
```

Comment:
Should all those types come from a "models" directory?

Reply:
We are gonna move them after we release the v3 :) 
Thank you for the feedback.